components/generate/layer/LayerAssetPreviewCard.tsx

```diff
@@ -66,58 +70,72 @@ function LayerAssetPreviewCard({ asset, layerIndex, assetIndex }: Props) {
       initial="rest"
       style={{ position: "relative" }}
     >
-      <motion.div
-        variants={buttonVariant}
-        style={{ position: "absolute", top: 10, right: 10, zIndex: 100 }}
-      >
-        <IconButton
-          onClick={() => removeAsset(layerIndex, assetIndex)}
-          size="small"
-          aria-label="Delete asset"
-          component="label"
-          sx={{
-            backgroundColor: "white",
-            border: 1,
-            borderColor: "black",
-            "&:hover": {
-              backgroundColor: "white"
-            }
-          }}
-        >
-          <GridCloseIcon fontSize="small" sx={{ color: "black" }} />
-        </IconButton>
-      </motion.div>
       <motion.div variants={variant}>
         <Box
           sx={{
             position: "relative",
-            height: "220px",
+            maxHeight: "240px",
             width: "160px",
             overflow: "hidden",
-            border: 2
+            border: 2,
+            marginRight: 2,
+            marginBottom: 2,
+            borderRadius: 2
           }}
         >
           <Stack>
-            <PNGAssetPreview
-              name={asset.name}
-              data={asset.data}
-              width={160}
-              height={160}
-              showPngGrid
-            />
-            <Box sx={{ height: "60px", padding: 2 }}>
+            <Box p={2} sx={{ display: "grid", placeItems: "center" }}>
+              <PNGAssetPreview
+                name={asset.name}
+                data={asset.data}
+                width={120}
+                height={120}
+                showPngGrid
+                sx={{ border: 2 }}
+              />
+            </Box>
+            <Box sx={{ borderBottom: 2 }} />
+
+            <Box
+              sx={{ padding: 2, backgroundColor: grey[300], height: "auto" }}
+            >
               <motion.div variants={textVariant}>
                 <Typography
                   component="p"
```

Comment:
`<Typogrpahy />` uses paragraph element as a default. No need to declare menually.